### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by drcivan**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

_Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request._

Please provide enough information so that others can review your pull request:

Explain the **details** for making this change. What existing problem does the pull request solve?

### Test plan (required)

Demonstrate the code is solid. If not, please add `WIP:` in its title.

### Closing issues

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### Self Check before Merge

Please check all items below before review.

-   [ ] Corresponding issues are created/updated or not needed
-   [ ] Docs are updated/provided or not needed
-   [ ] Examples are updated/provided or not needed
-   [ ] TypeScript definitions are updated/provided or not needed
-   [ ] Tests are updated/provided or not needed
-   [ ] Changesets are provided or not needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,125 +37,82 @@
...
-## About
-
-[**refine**](https://refine.dev/) offers lots of out-of-the box functionality for rapid development, without compromising extreme customizability. Use-cases include, but are not limited to _admin panels_, _B2B applications_ and _dashboards_.
-
-## Documentation
-
-For more detailed information and usage, refer to the [refine documentation](https://refine.dev/docs/).
 
 ## What is refine?
+**refine** is a React-based framework for rapid ✨ development of web applications.
```
Comment (necatiozmen) [inline README.md:48]:
```suggestion
**refine** is a React-based framework for the rapid ✨ development of web applications. 
```

File: README.md [hunk 2]
```diff
@@ -37,125 +37,82 @@
...
-
-[**refine**](https://refine.dev/) offers lots of out-of-the box functionality for rapid development, without compromising extreme customizability. Use-cases include, but are not limited to _admin panels_, _B2B applications_ and _dashboards_.
-
-## Documentation
-
-For more detailed information and usage, refer to the [refine documentation](https://refine.dev/docs/).
 
 ## What is refine?
+**refine** is a React-based framework for rapid ✨ development of web applications. 
+It eliminates the repetitive tasks demanded by **CRUD** operations and provides industry standard solutions for critical parts like **authentication**, **access control**, **routing**, **networking**, **state management** and **i18n**.
```
Comment (necatiozmen) [inline README.md:49]:
```suggestion
It eliminates the repetitive tasks demanded by **CRUD** operations and provides industry standard solutions for critical parts like **authentication**, **access control**, **routing**, **networking**, **state management**, and **i18n**.
```

File: packages/mui/README.md [hunk 3]
```diff
@@ -35,228 +35,178 @@
...
 
 ## Quick Start
 
-Run the **superplate** tool with the following command:
+The fastest way to get started with **refine** is using the [superplate](https://github.com/pankod/superplate) project starter tool.
+Run the following command to create a new **refine** project configured with  [Ant Design System](https://ant.design/) as the default UI framework:
 
 ```
-npx superplate-cli -o refine-mui tutorial
+npx superplate-cli --preset refine-antd my-project
```
Comment (aliemir) [inline packages/mui/README.md:91]:
```suggestion
npx superplate-cli --preset refine-antd my-project
```

Wrong framework here

File: packages/mui/README.md [hunk 4]
```diff
@@ -35,228 +35,178 @@
...
-<div align="center">
-    <img src="https://raw.githubusercontent.com/pankod/refine/master/documentation/static/img/getting-started/benchmark.png" width="400px" align="center" />
-</div>
+✅ Full **test coverage**
 
 ## Quick Start
 
-Run the **superplate** tool with the following command:
+The fastest way to get started with **refine** is using the [superplate](https://github.com/pankod/superplate) project starter tool.
+Run the following command to create a new **refine** project configured with  [Ant Design System](https://ant.design/) as the default UI framework:
```
Comment (aliemir) [inline packages/mui/README.md:88]:
```suggestion
Run the following command to create a new **refine** project configured with  [Ant Design System](https://ant.design/) as the default UI framework:
```

Wrong framework here as well

File: packages/mui/README.md (hunk 3 above)
Comment (omeraplak) [reply packages/mui/README.md:91]: `npx superplate-cli -o refine-antd my-project`

**Comment by necatiozmen** [comment]
Headline will looks nicer if <br />  will be added above the refine logo

File: packages/antd/README.md [hunk 5]
```diff
@@ -214,23 +175,38 @@ export const PostList: React.FC = () => {
...
 
+Now, you should see the output as a table populated with `post` & `category` data:
+![First example result](https://github.com/pankod/refine/blob/master/documentation/static/img/first-example-result.png?raw=true)
+## Next Steps
+
+👉 Jump to [Refine<>Ant Design Tutorial](https://refine.dev/docs/ui-frameworks/antd/tutorial/) to continue your work and turn the example into a full-blown CRUD application.
+
+👉 Check out the [Refine<>Tailwind Tutorial](https://refine.dev/docs/ui-frameworks/antd/tutorial/) to learn how to use **refine** in a pure *headless* way.
+
+👉 Visit [Learn the Basics Page](https://refine.dev/docs/getting-started/basics/) to get informed about the fundemental concepts.
```
Comment (ozkalai) [inline packages/antd/README.md:200]:
```suggestion
👉 Visit [Learn the Basics Page](https://refine.dev/docs/getting-started/basics/) to get informed about the fundamental concepts.
```

**Pull request merged**